plugins/pencilblue/templates/angular/admin/plugins/manage_plugins.html

```diff
@@ -105,6 +104,17 @@
 			doJobRetrieve();
 		};
 
+		$scope.isInstalled = function(pluginName) {
+			var result = $.grep($scope.installedPlugins, function(currentPlugin){
```

Comment:
Let's find a way to accomplish this with the jQuery lite in Angular, instead of injecting full jQuery into an Angular controller.